sdk/spring/azure-spring-boot/src/main/resources/azure-spring-actuator.properties

```diff
@@ -1,2 +0,0 @@
-management.health.azure-cosmos.enabled=false
-management.endpoint.health.show-details=always
```

Comment:
can the file "azure-spring-actuator.properties" be removed?

Reply:
Yes, because we can do without this configuration file and still work fine.